src/content/docs/contributing/nf_core_basic_training.md

```diff
@@ -76,64 +76,276 @@ To save any file from the explorer panel, right-click the file and select Downlo
+│   │   ├── mqc_fastqc_counts.png
+│   │   ├── mqc_fastqc_quality.png
+│   │   ├── nf-core-demotest_logo_dark.png
+│   │   └── nf-core-demotest_logo_light.png
+│   ├── output.md
+│   ├── README.md
+│   └── usage.md
+├── lib
+│   ├── nfcore_external_java_deps.jar
+│   ├── NfcoreTemplate.groovy
+│   ├── Utils.groovy
+│   ├── WorkflowDemotest.groovy
+│   └── WorkflowMain.groovy
+├── LICENSE
+├── main.nf
+├── modules
+│   ├── local
+│   │   └── samplesheet_check.nf
+│   └── nf-core
+│       ├── custom
+│       │   └── dumpsoftwareversions
+│       │       ├── main.nf
+│       │       ├── meta.yml
+│       │       └── templates
+│       │           └── dumpsoftwareversions.py
+│       ├── fastqc
+│       │   ├── main.nf
+│       │   └── meta.yml
+│       └── multiqc
+│           ├── main.nf
+│           └── meta.yml
+├── modules.json
+├── nextflow.config
+├── nextflow_schema.json
+├── pyproject.toml
+├── README.md
+├── subworkflows
+│   └── local
+│       └── input_check.nf
+├── tower.yml
+└── workflows
+    └── demotest.nf
+```
+
+These are the files in detail:
+
+1. **main.nf**
+
+   This file contains the main nextflow pipeline code. Mostly this file is not touched.
+
+2. **workflows/demotest.nf**
+
+   This file is where the pipeline is going to be assembled. It connects the different modules and subworkflows.
+
+3. **CHANGELOG.md, CODE_OF_CONDUCT.md, LICENSE, README.md, CITATIONS.md**
+
+   These are standard files created for github repositories. As a default, this pipeline will be under an MIT licence. The CODE_OF_CONDUCT is specific for the nf-core community.
+
+4. **assets/**
+
+   This directory contains different templates such as email templates or the MultiQC config. In this course contents of this directory can be largely ignored.
+
+5. **bin/**
+
+   The `bin` directory contains executable scripts which are automatically added to the path by nextflow. As such, they can be called by name without using their absolut or relativ path. The python script `check_samplesheet.py` is part of the nf-core template, since typically, nf-core pipelines require a samplesheet as one of their inputs.
+
+6. **conf/** and **nextflow.config**
+
+   The `nextflow.config` file is the main config file, it includes all the configurations in `conf/`. Importantly, `conf/` contains the `test.config` file, which is used for pipeline testing. In this course we are not going to touch config files, but they have been extensively covered in the following bytesize talks: [How nf-core configs work (nf-core/bytesize #2)](https://www.youtube.com/watch?v=cXBYusdjrc0&list=PL3xpfTVZLcNiF4hgkW0yXeNzr0d35qlIB&index=8&pp=gAQBiAQB), [Making a new institutional config profile (nf-core/bytesize #10)](https://www.youtube.com/watch?v=Ym1s6sKGzkw&list=PL3xpfTVZLcNiF4hgkW0yXeNzr0d35qlIB&index=9&pp=gAQBiAQB), [nf-core/bytesize: Using nf-core configs in custom pipelines](https://www.youtube.com/watch?v=zgcrI_0SUgg&list=PL3xpfTVZLcNiF4hgkW0yXeNzr0d35qlIB&index=40&pp=gAQBiAQB)
```

Comment:
```suggestion
   The `nextflow.config` file is the main config file. In addition to supplying default parameters, it imports all the configurations in `conf/`. Importantly, `conf/` contains the `test.config` file, which is used for pipeline testing. In this course we are not going to touch config files, but they have been extensively covered in the following bytesize talks: [How nf-core configs work (nf-core/bytesize #2)](https://www.youtube.com/watch?v=cXBYusdjrc0&list=PL3xpfTVZLcNiF4hgkW0yXeNzr0d35qlIB&index=8&pp=gAQBiAQB), [Making a new institutional config profile (nf-core/bytesize #10)](https://www.youtube.com/watch?v=Ym1s6sKGzkw&list=PL3xpfTVZLcNiF4hgkW0yXeNzr0d35qlIB&index=9&pp=gAQBiAQB), [nf-core/bytesize: Using nf-core configs in custom pipelines](https://www.youtube.com/watch?v=zgcrI_0SUgg&list=PL3xpfTVZLcNiF4hgkW0yXeNzr0d35qlIB&index=40&pp=gAQBiAQB)
```
Although includeConfig is the keyword in the config, I think imports is more understandable to the reader.